packages/models/src/lib/category-config.ts

```diff
@@ -8,6 +8,8 @@ import {
 } from './implementation/schemas';
 import { errorItems, hasDuplicateStrings } from './implementation/utils';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type HackForCyclicRefs = any;
```

Comment:
I guess a safer way around this would be to duplicate the type (you only need to include what the functions use), that way you don't lose type-safety.

```suggestion
type RefsList = {
  ref: string
}[];
```

Then use `metrics: RefsList` as function argument below.